airflow/utils/db.py

```diff
@@ -0,0 +1,204 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+from functools import wraps
+import logging
+import os
+
+from alembic.config import Config
+from alembic import command
+from alembic.migration import MigrationContext
+
+from sqlalchemy import event, exc
+from sqlalchemy.pool import Pool
+
+from ..airflow import settings
+from ..airflow import configuration
```

Comment:
Could we make these absolute imports? http://stackoverflow.com/questions/4209641/absolute-vs-explicit-relative-import-of-python-module